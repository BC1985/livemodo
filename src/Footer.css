footer {
  display: flex;
  justify-content: center;
  align-items: center;
  position: sticky;
  left: 0;
  bottom: 0;
  width: 100%;
  background-color: rgb(65, 63, 63);
  height: 50px;
  color: white;
  text-align: center;
}
