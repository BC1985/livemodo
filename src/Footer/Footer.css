.footer {
  width: 100%;
  height: 30px;
  padding: 5px;
  background: rgb(56, 53, 63);
  color: rgb(214, 212, 207);
  text-align: center;
  position: sticky;
  font-weight: lighter;
  bottom: 0;
  left: 0;
}
