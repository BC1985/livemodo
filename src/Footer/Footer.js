import React from "react";
import "./Footer.css";

export default function Footer() {
  return (
    <div>
      <footer className="footer">
        &#169;All rights reserved to Ben Cohen 2019
      </footer>
    </div>
  );
}
