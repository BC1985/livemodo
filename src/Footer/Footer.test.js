import React from "react";
import ReactDOM from "react-dom";
import Footer from "../Footer/Footer";
import renderer from "react-test-renderer";

it("renders without crashing", () => {
  const div = document.createElement("div");
  ReactDOM.render(<Footer />, div);
  ReactDOM.unmountComponentAtNode(div);
});

it("renders UI as expected", () => {
  const tree = renderer.create(<Footer />).toJSON();
  expect(tree).toMatchSnapshot();
});
