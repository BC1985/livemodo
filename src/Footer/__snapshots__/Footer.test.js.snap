// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders UI as expected 1`] = `
<div>
  <footer
    className="footer"
  >
    Â©All rights reserved to Ben Cohen 2019
  </footer>
</div>
`;
